src/sentry/consumers/__init__.py

```diff
@@ -269,6 +269,14 @@ def ingest_events_options() -> list[click.Option]:
             "consumer_type": "events",
         },
     },
+    "ingest-feedbacks": {
+        "topic": settings.KAFKA_INGEST_FEEDBACKS,
+        "strategy_factory": "sentry.ingest.consumer.factory.IngestStrategyFactory",
```

Comment:
Each consumer should have it's own strategy? (the ingest strategy is already doing far more than it should and we should not add additional logic to it)

Reply:
Any tips/references for how to write a new strategy? In ingest/consumer/factory.py, right

Reply:
The directory structure in sentry is a bit misleading, but the `ingest` module really just refers to the errors/transactions ingest consumer today and not any of the rest of the piplines. I would suggest keeping it either entirely separate, or inside a new module `ingest.user_feedback` or something like that. See https://getsentry.github.io/arroyo/ for docs how to write a strategy.

Reply:
@lynnagara We're not adding additional logic to it.  This behavior already exists in that consumer.  We're just moving it to a new topic.  I agree we should split out this product into its own consumer.  Can that be handled in a follow-up PR?